tests/components/ruckus_unleashed/__init__.py

```diff
@@ -48,25 +65,32 @@
 
 TEST_CLIENT_ENTITY_ID = "device_tracker.ruckus_test_device"
 TEST_CLIENT = {
-    API_IP: "1.1.1.2",
-    API_MAC: "AA:BB:CC:DD:EE:FF",
-    API_NAME: "Ruckus Test Device",
-    API_ACCESS_POINT: "00:11:22:33:44:55",
+    API_CLIENT_IP: "1.1.1.2",
+    API_CLIENT_MAC: "AA:BB:CC:DD:EE:FF",
+    API_CLIENT_HOSTNAME: "Ruckus Test Device",
+    API_CLIENT_AP_MAC: DEFAULT_AP_INFO[0][API_AP_MAC],
 }
 
+DEFAULT_TITLE = DEFAULT_MESH_INFO[API_MESH_NAME]
```

Comment:
As the test's default title changed, I only double check...
Do you know if the mesh names of the old library and the new one are the same?

Reply:
1. Yes, the mesh names are consistent between the old and new libraries.
2. I just updated other mocks and registry entries so that hopefully everything should now match the previous values.